calico-cloud_versioned_docs/version-3.17/threat/web-application-firewall.mdx

```diff
@@ -67,253 +67,226 @@ Enabling WAF for certain system services may result in an undesired cluster stat
+metadata:
+  name: tigera-secure
+spec:
+  webApplicationFirewall: Enabled
+EOF
+```
 
-To simulate an attack to view the default warning in Service Graph, follow these steps:
+## Apply WAF
 
-1. Get the cluster/service URL.
-   - The external address of your cluster/service
-   - The cluster IP of your application's service (if testing within the cluster)
-2. Enter the following command to trigger a SQL injection attack.
+Now that your cluster is enabled for WAF, you can apply WAF to your services and make sure that your
+Web apps are protected from application layer attacks. 
 
-##### Example: SQL Injection attack
+If you have deployed the sample application, you can apply WAF on a service associated with your app, as follows:
 
+```bash
+kubectl annotate svc frontend -n default --overwrite projectcalico.org/l7-logging=true
 ```
-  curl http://<host>//test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user
-```
+Alternatively, you can [use the Manager UI](#apply-waf-to-your-services) to apply WAF to the `frontend` service.
 
-In Service Graph, go to the service and click the HTTP tab. You will see the following warning:
+In this example, we applied WAF to the `frontend` service. This means that every traffic that goes through the `frontend` service is inspected.
+However, the traffic is not blocked because the WAF rule is set to `DetectionOnly` by default.  
+When you are convinced that the traffic for the service to be denied or dropped, you can [set it to block](#set-waf-rule-to-block-traffic).
 
-```
-  level=warning msg="WAF Process Http Request [2f435aca-4a3e-4f96-a7b2-a60c9745bd7b] URL '/test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user' OWASP Warning'[2] Host:'10.224.0.38' File:'/etc/modsecurity-ruleset/REQUEST-942-APPLICATION-ATTACK-SQLI.conf' Line:'45' ID:'942100' Data:'' Severity:'0' Version:'OWASP_CRS/3.3.2' Message:'Warning. detected SQLi using libinjection.''"
-```
+### Apply WAF to your services
 
-#### Kibana
+In the previous example, we applied WAF to the `frontend` service of the sample application. Here, we are
+applying WAF to a service of your own application.
 
-To view WAF events In Kibana:
-
-1. Create the `tigera_secure_ee_waf*` index pattern. For help with index patterns, see [Kibana index pattern] (../visibility/kibana.mdx).
-2. Select the `tigera_secure_ee_waf*` index pattern. 
+1. On the Manager UI, click **Threat Defense**, **Web Application Firewall**.
+2. Select the services you want WAF to inspect, and then click **Confirm Selections**.
+    
+   <img
+    src='/img/calico-enterprise/waf-select-services-3.17.png'
+    alt='WAF services'
+    width='600'
+  />
 
-You should see the relevant WAF assessment from your request.
+3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
 
-### Create global alerts
```

Comment:
Adding/Creating the Global Alerts doesn't need to be removed for 3.17. Just from 3.18 onwards :+1:  Sorry for the confusion

Reply:
Reverted.